core/environment/transition_reset.go

```diff
@@ -53,19 +53,15 @@ func (t ResetTransition) do(env *Environment) (err error) {
 						task.RESET.String(),
 						task.STANDBY.String(),
 						nil,
+						env.Id(),
 					)
 	t.taskman.MessageChannel <- taskmanMessage
 
-	// err = t.taskman.TransitionTasks(
-	// 	env.Workflow().GetTasks(),
-	// 	task.CONFIGURED.String(),
-	// 	task.RESET.String(),
-	// 	task.STANDBY.String(),
-	// 	nil,
-	// )
-	// if err != nil {
-	// 	return
-	// }
+	incomingEv := <-env.stateChangedCh
```

Comment:
Same comment as above, do we need a select/timeout here to ensure we don't stay blocked forever?